rbac/management/auditLogs/model.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Model for audit logging."""
+from django.db import models
+
+
+class AuditLogModel(models.Model):
+    """An audit log."""
+
+    GROUP = "group"
+    ROLE = "role"
+    USER = "user"
+    PERMISSION = "permission"
+    RESOURCE_CHOICES = (
+        (GROUP, "Group"),
+        (ROLE, "Role"),
+        (USER, "User"),
+        (PERMISSION, "Permission"),
+    )
+
+    DELETE = "delete"
+    ADD = "add"
+    EDIT = "edit"
+    CREATE = "create"
+    REMOVE = "remove"
+    ACTION_CHOICES = (
+        (DELETE, "Delete"),
+        (ADD, "Add"),
+        (EDIT, "Edit"),
+        (CREATE, "Create"),
+        (REMOVE, "Remove"),
+    )
+
+    date = models.DateField(auto_now_add=True)
+    requester = models.TextField(max_length=255, null=False)  # probs need a max length allowed here
+    description = models.TextField(max_length=255, null=False)  # probs need a max length allowed here
+    resource = models.CharField(max_length=32, choices=RESOURCE_CHOICES)
+    action = models.CharField(max_length=32, choices=ACTION_CHOICES)
+
+    def create_data(self, request, resource):
```

Comment:
I think that name could be just `create` it implies to create one record in Audit Log table.

Reply:
OK, I misunderstood it - you are basically creating log records which would not makes in update/delete method. So maybe it would be better to have word as suffix. But I am thinking about names of methods - name should represent fact that:

I am creating log record about create/update/delete action, naming is hard

Reply:
all methods are going to create log record so maybe it could something like 
log_create_action
log_delete_action 

not sure exactly,  I am definitely interested in your ideas! 💡

Reply:
Maybe we can have it even shorter like: 
log_create
log_delete? 
I do like having the word log in front to make it clear!